.github/workflows/docker_autometa.yml

```diff
@@ -50,6 +52,7 @@ jobs:
             type=raw,value=latest,enable=${{ endsWith(github.ref, github.event.repository.default_branch) }}
             type=raw,value={{branch}}
             type=semver,pattern={{version}}
+            type=schedule,pattern=nightly
```

Comment:
This will tag the image as `docker-image:nightly` (from the [docker/metadata-action docs](https://github.com/docker/metadata-action#typeschedule))